apps/system/style/window.css

```diff
@@ -27,7 +27,9 @@
   pointer-events: none;
 }
 
-.appWindow.attentionWindow .render {
```

Comment:
wait wait wait :)
we need this line, otherwise you can have any interaction in the attention window (try hanging up a call)